recipes/ubermag/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "ubermag" %}
+{% set version = "0.1" %}
+{% set sha256 = "8cacbb83f1f3c61338194ae83029c06b54ec9f4a10cb9ec365cccac16b42f0d4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: pip install --no-deps . -vvv
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - oommfc
+    - pytest
+
+test:
+  imports:
+    - ubermag
+
+  commands:
```

Comment:
```suggestion
  requires:
    - pytest
  commands:
```

Reply:
Thank you @marcelotrevisani, we also need `pytest` as a dependency to run our package because we provide
```
import ubermag
ubermag.test()
```
functionality.